cert_tree.py

```diff
@@ -187,10 +187,18 @@ def main():
 
     args = parser.parse_args()
     cert_file = args.cert_file
-    if not cert_file.endswith('pem'):
+
+    # Check if the cert is in the right format
+    proc = subprocess.run(['openssl', 'x509', '-noout', 'in', cert_file], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+    print(proc.returncode)
```

Comment:
Printing return code every time is not needed, I think. Do you agree?

Reply:
> 
> 
> Printing return code every time is not needed, I think. Do you agree?

Yes, of course. I just forget to remove this line. Sorry.